_pages/btaylor19892.yaml

```diff
@@ -0,0 +1,5 @@
+---
+githubHandle: btaylor1989
+pageUrl: https://btaylor1989.github.io/command-bt/
+timestamp: 2017-02-19
+---
```

Comment:
This line should be followed by a "new line" by adding `enter` or `return` after `---`. 
This file should have 6 lines in total.